### PR TITLE
feat(comp): Rewrite Columns and remove Column

### PR DESCRIPTION
**Pull request by ti10le**
#1860

### CODE REVIEW

File: .changeset/tidy-eggs-try.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+"@marigold/docs": major
```
Comment (sebald) [inline .changeset/tidy-eggs-try.md:2]: If we already do this, the next release will be a major. Ist das okay?

Comment (ti10le) [reply .changeset/tidy-eggs-try.md:2]: Oh yes sure. I thought just about that is a breaking change.

Comment (sebald) [reply .changeset/tidy-eggs-try.md:2]: Until we make a 1.0 it's fine to do a minor. We did this until now 🤠

File: packages/components/src/Columns/Columns.tsx [hunk 2]
```diff
@@ -1,69 +1,55 @@
...
-              <Box css={{ p: `${spaceValue / 2}px` }}>
-                {child.props.children}
-              </Box>
-            );
-          }
-        )}
-      </Box>
+    <Box
+      display="flex"
+      __baseCSS={{
```
Comment (sebald) [inline packages/components/src/Columns/Columns.tsx:41]: put everything into `css` or `__baseCSS`. makes it hard to read when it is spread over 3 options

**Review by sebald — CHANGES_REQUESTED** [review]
great work, just some minor fixes needed :)

**Review by sebald — APPROVED** [review]

**Pull request merged**